lib/morse_translator.rb

```diff
@@ -40,4 +43,23 @@ def initialize
                     " " => " "}
   end
 
+  def translate(words)
+    downcase = words.downcase
+    new_words = ""
+    downcase.each_char do |letter|
+      new_words << @dictionary[letter]
+    end
+    new_words
+  end
+
+  def morse_to_eng(morse)
+      morse_chars = morse.split(" ")
+      reverse_dictionary = @dictionary.invert
+      english = ""
+      morse_chars.each do |sequence|
+        english << reverse_dictionary[sequence]
+      end
+      english
+  end
```

Comment:
Close! Doesn't take account of the double space between words in the morse code input being translated to a single space in the English output.